exercises/dnd-character/description.md

```diff
@@ -0,0 +1,30 @@
+For a game of Dungeons & Dragons, you generate your character's six ability
```

Comment:
I was wondering if we should perhaps link to the Wiki page of D&D: https://en.wikipedia.org/wiki/Dungeons_%26_Dragons, which might help people not familiar with D&D to become acquainted with it.

Reply:
Good idea.